src/amo/components/Home/styles.scss

```diff
@@ -114,11 +128,26 @@
   height: 64px;
   margin-bottom: 5px;
   transform: scale(0.8);
-  width: 100%;
+  width: 35%;
 
   @include respond-to(medium) {
     transform: initial;
   }
+
+  @include respond-to(large) {
+    background-position: center;
+    width: 100%;
+  }
+}
+
+*[dir='rtl'] {
+  .Home-SubjectShelf-list-item .Icon {
+    background-position: right;
+    unicode-bidi: embed;
+    @include respond-to(large) {
+      background-position: center;
```

Comment:
hi @hritvi, can we insert this above (and use sass' nesting here)?

```css
.Home-SubjectShelf-list-item .Icon {
   ...
   @include respond-to(large) {
    background-position: center;
    width: 100%;
  }
  
  // insert this here 
  [dir='rtl'] & {
    background-position: right;
    // unicode-bidi: embed;
    @include respond-to(large) {
      background-position: center;
    }
  }
}

technically what you have is right but I think we should take advantage of sass' nesting since it can be done pretty easily here.

Reply:
I suppose we can't do that because the <code>dir</code> tag is an attribute of HTML and not the span with the class Home-SubjectShelf-list-item. So we can't include it in class

I tried 
```
@include respond-to(large) {
    background-position: center;
    width: 100%;
  }
  html[dir=rtl] & {
    .Home-SubjectShelf-list-item .Icon {
      background-position: right;
      @include respond-to(large) {
        background-position: center;
      }
    }
```
But browser is not able to render the CSS written in the [dir='rtl'] block.

Reply:
hi @hritvi, it seems that you re-nested the `.Home-SubjectShelf-list-item .Icon` again which will make it not work...

so you want to have something like this:

```diff
--- a/src/amo/components/Home/styles.scss
+++ b/src/amo/components/Home/styles.scss

.Home-SubjectShelf-list-item .Icon {
  background-position: left;
  background-repeat: no-repeat;
  background-size: 64px;
  border-radius: $border-radius-default;
  content: '';
  display: block;
  height: 64px;
  margin-bottom: 5px;
  transform: scale(0.8);
  width: 35%;

  @include respond-to(medium) {
    transform: initial;
  }

  @include respond-to(large) {
    background-position: center;
    width: 100%;
  }

+  [dir='rtl'] & {
+    background-position: right;
+
+    @include respond-to(large) {
+      background-position: center;
+    }
+  }
}

- *[dir='rtl'] {
-  .Home-SubjectShelf-list-item .Icon {
-    background-position: right;
-    unicode-bidi: embed;
-    @include respond-to(large) {
-      background-position: center;
-    }
-  }

...
```
Can you try that? I think it's just a matter of fixing the nesting issue...
Let me know how that works for you :)

Reply:
or let me know if any questions :)

Reply:
Yes, it worked. I have updated the PR. Thanks :smile: